hw2/05-form.html

```diff
@@ -7,14 +7,94 @@
     <main>
-      <form>
-        <h1>Exercise 05 - Form</h1>
-      </form>
+      <section class="mx-auto p-3" style="width: 33em">
+        <div class="card p-3 mb-2 bg-light text-dark">
+          <form id="form">
+            <h1 class="mb-4 text-center">Exercise 05 - Form</h1>
+
+            <div class="mb-3">
+              <label for="userName" class="form-label fw-bold">Full Name:</label>
+              <input type="text" class="form-control" id="userName" />
+            </div>
+
+            <div class="mb-3">
+              <label for="userEmail" class="form-label fw-bold">Email:</label>
+              <input type="email" class="form-control" id="userEmail" />
+            </div>
+
+            <div class="mb-3">
+              <label for="regStatus" class="form-label fw-bold"
+                >Registration Status:
+              </label>
+              <input
+                class="form-control"
+                list="datalistOptions"
+                id="regStatus"
+                placeholder="Choose an option"
+              />
+              <datalist id="regStatusOptions">
+                <option value="Registered"></option>
+                <option value="Not Registered"></option>
+                <option value="Withdrawn"></option>
+                <option value="Dropped"></option>
+                <option value="Other"></option>
+              </datalist>
+            </div>
+            <div class="mb-3">
+              <label class="mb-2 fw-bold" for="courses"">Have you taken any of the following courses? </label>
+              <div class="form-check">
+                <input
+                  class="form-check-input"
+                  type="checkbox"
+                  id="progLang"
+                />        
+                <label class="form-check-label" for="progLang">
+                  Programming Languages
+                </label>
+              </div>
+              <div class="form-check">
+                <input
+                  class="form-check-input"
+                  type="checkbox"
+                  id="opSys"
+                />
+                <label class="form-check-label" for="opSys">
+                  Operating Systems
+                </label>
+              </div>
+              <div class="form-check">
+                 <input
+                  class="form-check-input"
+                  type="checkbox"
+                  id="fsWebDev"
+                />
+                <label class="form-check-label" for="fsWebDev">
+                  Full Stack Web Development
+                </label>
+                </div>
+              </div>
+           
+            <div class="mb-3">
+              <label for="otherInfo" class="form-label fw-bold">Anything Else?</label>
+              <textarea class="form-control" id="otherInfo" rows="3"></textarea>
+            </div>
+          
+            <input id="submitBtn" type="submit" class="btn btn-primary btn-block" value="Submit" />
```

Comment:
Instead of using the `input` element, consider using `button`.  It is more semantic and it's accessible for screen reader.